features/custom_error_pages.feature

```diff
@@ -0,0 +1,16 @@
+@allow-rescue
+@allow-error-pages
```

Comment:
By default Cucumber surfaces exceptions to ease debugging. These tags trigger hooks that allow production-like exception handling behaviour (i.e. friendly error pages).